samples/jsonParserComparison.js

```diff
@@ -0,0 +1,189 @@
+const snowflake = require('snowflake-sdk');
+const helpers = require('./helpers');
+const blocked = require('blocked-at');
+
+async function run() {
+  const defaultRowCount = 300000;
+  const rowCountToInsert = process.argv[2];
+  const rowCountToFetch = process.argv[3];
+  const choosenParser = process.argv[4];
+  console.log('Started with arguments: ');
+  console.log(`Inserted rows amount: ${rowCountToInsert} - default ${defaultRowCount}`);
+  console.log(`Selected rows amount: ${rowCountToFetch} - default ${defaultRowCount}`);
+  console.log(`Selected json parse : ${choosenParser} - default all of Function, eval, better-eval, JSON`);
```

Comment:
we should not have `eval` here and we should have `vm` instead

and `betterEval` should be camel cased?